### PR TITLE
Update README.md for sample_spl2_pii_masking app

### PR DESCRIPTION
**Pull request by lstewartsplunk**
Updated to change how to access the app. Users don't need to install the app.

### CODE REVIEW

**Review by PKing70 — COMMENTED** [review]
Should be https://github.com/splunk/splunk-app-examples/tree/master/spl2-sample-apps/sample_spl2_pii_masking

ppi to pii

**Review by tdhellmann — APPROVED** [review]

**Comment by lstewartsplunk** [comment]
@PKing70 - Fixed the title

**Comment by lstewartsplunk** [comment]
Ready to merge

File: spl2-sample-apps/sample_spl2_pii_masking/README.md [hunk 1]
```diff
@@ -26,7 +26,13 @@ This app is compatible with Splunk Enterprise.
 
 You need a pre-release version of Splunk Enterprise that supports SPL2-based applications. See [SPL2 Public Beta overview](https://dev.splunk.com/enterprise/docs/developapps/createspl2apps/spl2previewoverview) for information about the Beta.
 
-## Installation
+## Accessing the sample app
+
+The app is available in the [spl2-sample-apps](https://github.com/splunk/splunk-app-examples/tree/master/spl2-sample-apps/sample_spl2_ppi_masking) folder on the Splunk GitHub Examples repository.
```
Comment (PKing70) [inline spl2-sample-apps/sample_spl2_pii_masking/README.md:31]: https://github.com/splunk/splunk-app-examples/tree/master/spl2-sample-apps/sample_spl2_pii_masking

**Pull request merged**